data/categories.json

```diff
@@ -63,5 +63,8 @@
     "scourge":  [
       "Midnight Smith"
     ]
-  }
+  },
+  "exceptions": [
+      ["860688654", "Season 7 'Intrepid' armor sets"]
+  ]
```

Comment:
can we change this to an object?
```
exceptions: {
 "8606888654": "Season 7 'Intrepid' armor sets"
}
```
or does it work better working with an array of arrays?